firmware/application/apps/ui_freqman.cpp

```diff
@@ -25,12 +25,12 @@
 #include "portapack.hpp"
 #include "event_m0.hpp"
 
+static int32_t current_category_id = 0;
```

Comment:
nit: why not keep this in the class so it's not a global?

Reply:
I moved it back to the ui_freqman.cpp inside the namespace, since it's still working that the best I can do.